docs/source/reference/benchmark/cli.rst

```diff
@@ -0,0 +1,81 @@
+.. _benchmark-cli:
+
+CLI
+=============
+
+Workflow
+--------------------------------
+
+You can use SkyPilot Benchmark by simply replacing your ``sky launch`` command with ``sky bench launch``:
+
+.. code-block:: bash
+
+    # Normal launch command
+    $ sky launch mytask.yaml --gpus V100
```

Comment:
Maybe we should remove this to make the code block more copy paste friendly

Reply:
Good point. Users of this feature will be familiar to SkyPilot basic APIs, so we can skip that.